script.js

```diff
@@ -1,4 +1,84 @@
+var gameMode = 'waiting for user to press submit';
+var currentPlayer = 1;
+var currentPlayerRolls = [];
+var bothPlayersScores = [];
+
+var rollDice = function () {
+  var randomDecimal = Math.random() * 6;
+  var randomInteger = Math.floor(randomDecimal);
+  var diceNumber = randomInteger + 1;
+  return diceNumber;
+};
+
+var rollDiceForPlayer = function () {
+  var counter = 0
+  while (counter < 2) {
+    currentPlayerRolls.push(rollDice());
+    counter += 1;
+  }
+  return "Welcome Player " + currentPlayer + ". <br> You rolled " + currentPlayerRolls[0] + " for Dice 1 and " + currentPlayerRolls[1] + " for Dice 2. <br> Please choose which dice you would like to represent the first digit of your number."
+};
+
+var getPlayerResult = function (input) {
+  var playerResult;
+  if (input != 1 && input != 2) {
+    gameMode = "waiting for user to press submit";
+    return "Error! Please only enter 1 or 2, depending on which dice you would like to choose as the first digit of your final result. <br> You rolled " + currentPlayerRolls[0] + " for Dice 1 and " + currentPlayerRolls[1] + " for Dice 2.";
```

Comment:
You'll also need to break out of the main function's code sequence if the user's input is invalid (i.e., to make a return statement in the main function after line 66 and before your 'if' statement at line 67). Right now, the code continues to run because the condition still satisfy line 67 (i.e., currentPlayer == 1). 

There a few ways to do so - one way is to create a variable which tracks if the input is invalid (e.g., var invalidInput = true), and have an 'if' statement between line 66 & 67 that catches this and returns an output. Declare this variable in the main function preferably, because you want it to 'reset' back to false each time the function runs. 

At line 26, You might also want to delete everything that comes after <br>